recipes/damask/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "damask" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://damask.mpie.de/pub/Download/Current/{{ name }}-{{ version }}.tar.xz
+  sha256: c3f487ad1487586fbde696b1402d4dda8c3463ad0710b001ee8e4203eee413c8
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux] 
+    - make
+    - cmake
+    - hdf5 =*=mpi_mpich_*
+    - fftw =*=mpi_mpich_*
+  host:
+    - python
+    - pip
+    - xorg-libxfixes  # [linux]
+    - petsc =3.10
+    - perl
+    - numpy
+    - vtk
+    - hdf5 =*=mpi_mpich_*
+    - fftw =*=mpi_mpich_*
+    # - h5py
+    - scipy
+  run:
+    - python
+    - petsc =3.10
+    - perl
+    - numpy
```

Comment:
Are you using the C API here? If you are you probably need pin_compatible. If not you probably don’t need it in host

Reply:
DAMASK is a Fortran code with python utilities for pre- and post-processing. Only these python utilities depend on numpy but there are not using the C API, just pure Python.